src/pkg/utils/io.go

```diff
@@ -59,8 +59,13 @@ type TextTemplate struct {
 }
 
 // MakeTempDir creates a temp directory with the zarf- prefix.
-func MakeTempDir() (string, error) {
-	tmp, err := os.MkdirTemp("", tmpPathPrefix)
+func MakeTempDir(basePath string) (string, error) {
+	if basePath != "" && InvalidPath(basePath) {
```

Comment:
```suggestion
	if basePath != "" {
```

`utils.CreateDirectory` already performs a `InvalidPath` check